superset/security/manager.py

```diff
@@ -1206,3 +1205,16 @@ def can_access_based_on_dashboard(datasource: "BaseDatasource") -> bool:
 
         exists = db.session.query(query.exists()).scalar()
         return exists
+
+    def is_user_admin(self) -> bool:
+        if g.user.is_anonymous:
+            from superset import conf
```

Comment:
I know that this pattern is used elsewhere in this file, but `current_app.config.get(...)` seems more Flask like.